deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/data/dashboard_saturation_tests.py

```diff
@@ -222,14 +267,22 @@ def convert_json_to_csv(args: Namespace, json_output: str) -> None:
         args (Namespace): Script parameters.
         json_output (str): Path and file name of the log.
     """
+    keys = ['counters', 'summaries', 'histograms']
+
+    with open(json_output) as f:
+        data = json.load(f)
+
     for type in args.type:
-        with open(json_output) as f:
-            data = json.load(f)
+        filtered_data = data[type]
 
-        csv_filename = gen_csv_filename(args.csv, type)
+        if isinstance(filtered_data, list):
+            filtered_data = merge_dictionaries(filtered_data)
 
-        df = pd.json_normalize(data[type])
-        df.to_csv(csv_filename, index=False)
+        for key in keys:
+            csv_filename = gen_csv_filename(args.csv, type, key)
+            filtered_data[key]['timestamp'] = datetime.now(timezone.utc).isoformat()
```

Comment:
To adapt the timestamp to the structure of the data analysis module:
```suggestion
            filtered_data[key]['timestamp'] = datetime.now(timezone.utc).strftime('%Y-%m-%d %H:%M:%S')